pipenv/cli/command.py

```diff
@@ -569,7 +569,10 @@ def run_open(state, module, *args, **kwargs):
     else:
         p = c.out.strip().rstrip("cdo")
     echo(crayons.normal("Opening {0!r} in your EDITOR.".format(p), bold=True))
-    edit(filename=p)
+    from ..core import inline_activate_virtual_environment
+    inline_activate_virtual_environment()
+    environment={"VIRTUAL_ENV":os.environ["VIRTUAL_ENV"]}
+    edit(filename=p, env=environment)
```

Comment:
```suggestion
    edit(filename=p)
```

Reply:
Thanks for the feedback I've updated the code according to these suggestions.

I'm still mulling over how to test the code as it doesn't seem like click tests their edit function either. In theory `Popen` should grab the current environment now but how should we verify if that's being passed to the editor? I was considering checking if the virtual environment is active when the `edit` command is called but that will not confirm if the editor ultimately receives those variables.